.github/workflows/2-dependabot-alerts.yml

```
@@ -55,18 +55,18 @@ jobs:
           fetch-depth: 0 # Let's get all the branches.
 
       # Verify the PR updated package.json.
-      - name: Check package.json
+      - name: Check package.json for minimist version other than 1.2.5
         uses: skills/action-check-file@v1
         with:
           file: "code/src/AttendeeSite/package.json"
-          search: "1.2.6"
+          search: "\"minimist\":\ \"\\^(?!1\\.2\\.5)(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)(?:-((?:0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?\""
```

```suggestion
          search: "\"minimist\":\\s*\"\\^(?!1\\.2\\.5)(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)(?:-((?:0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9][0-9]*|[0-9]*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?\""
```

I believe any number of spaces would work in this context 😂

---

It seems that " "(`&#x20;`), "\n"(`&#x0a;`), "\r"(`&#x0d;`), and "\t"(`&#x09;`) can be used as whitespaces.
https://www.json.org/json-en.html